layouts/rss.xml

```diff
@@ -16,8 +16,7 @@
     <title>{{ if eq  .Title  .Site.Title }}{{ .Site.Title }}{{ else }}{{ with .Title }}{{.}} on {{ end }}{{ .Site.Title }}{{ end }}</title>
     <link>{{ .Permalink }}</link>
     <description>Recent content {{ if ne  .Title  .Site.Title }}{{ with .Title }}in {{.}} {{ end }}{{ end }}on {{ .Site.Title }}</description>
-    {{- with .Site.Params.customFavicon -}}
-    <webfeeds:icon>{{ . | absURL }}</webfeeds:icon>
+    {{- with .Site.Params.siteLogoImage -}}
```

Comment:
Please also revert the changes here to keep the icon.

Reply:
ok, it's reverted. Please check, thank you!